.drone.star

```diff
@@ -2436,7 +2436,7 @@ def buildGithubCommentForBuildStopped(suite, alternateSuiteName):
         "image": "owncloud/ubuntu:20.04",
         "pull": "always",
         "commands": [
-            'echo "<details><summary>:boom: Acceptance tests <strong>%s</strong> failed. The build is cancelled...</summary>\\n\\n" >> %s/comments.file' % (alternateSuiteName, dir["web"]),
+            'echo "<summary>:boom: Acceptance tests <strong>%s</strong> failed. The build is cancelled...</summary>\\n\\n${DRONE_BUILD_LINK}/${DRONE_JOB_NUMBER}${DRONE_STAGE_NUMBER}/1\\n" >> %s/comments.file' % (alternateSuiteName, dir["web"]),
```

Comment:
@phil-davis could you take another look here? Details & Summary tags are supposed to go together, from what I see before the changes in this PR it just missed the closing `</details>` tags? Also, `'` and `"` escaping seems off in both cases

Reply:
There was never a closing `details` tag in this comment. It had been copied from the comments about screenshots, which has a whole bunch of code that fills `details` and puts a closing tag at the end.

Reply:
The single and double quotes do pair up. I think they work OK.